worldedit-core/src/main/java/com/fastasyncworldedit/core/command/tool/brush/StencilBrush.java

```diff
@@ -47,7 +49,12 @@ public void build(EditSession editSession, BlockVector3 center, Pattern pattern,
         int cutoff = onlyWhite ? maxY - minY : 0;
         final SolidBlockMask solid = new SolidBlockMask(editSession);
 
-        Location loc = editSession.getPlayer().getLocation();
+        Actor actor = editSession.getActor();
+        if (actor != null && !(actor instanceof Player)) {
+            throw new RuntimeException(); //TODO add message here.
```

Comment:
👀

Reply:
I... didn't look at all the changes fully myself yet